project/assets/new-evaluation.less

```diff
@@ -0,0 +1,91 @@
+.new-eval {
+    margin: auto;
+    margin-top: 5%;
+    width: 70%;
+}
+
+textarea {
+    width: 100%;
+    border: 0.5px solid black;
+}
+
+input[type=date],
+input[type=text] {
+    width: 30%;
+    display: inline-block;
+    outline: 0;
+    border-width: 0 0 2px;
+    border-color: rgb(166, 166, 166);
+}
+
+input[type=radio] {
+    border-color: rgb(166, 166, 166);
+}
+
+table {
+    border-collapse: collapse;
+    width: 100%;
+    margin-top: 0%;
+}
+
+.new-eval-button {
+    box-shadow: 10px 35px -9px rgba(204, 194, 204, 1);
+    line-height: 52px;
+    width: 13%;
+    margin-left: 90%;
+    margin-top: 4%;
+}
+
+.new-eval-button-skin {
+    background-color: rgb(51, 153, 255);
+    color: rgb(255, 255, 255);
+    border: none;
+    font-family: roboto;
+    font-size: 20px;
+}
+
+option,
+select {
+    font-size: x-large;
+    color: rgb(77, 77, 77);
+    width: 7%;
+    text-align: center;
+    margin: 1% 5% 0 4%;
+    display: inline-block;
+}
+
+label {
+    font-size: large;
+    display: inline-block;
+    color: rgb(77, 77, 77);
+    margin-left: -4%;
+    margin-right: 4%;
+    margin-top: 1%;
+    width: 15%;
+    text-align: left;
+    font-family: roboto;
+}
+
+legend {
+    color: rgb(51, 153, 255);
+    font-size: x-large;
+    font-weight: bold;
+}
+
+ul {
+    list-style-type: none;
+}
+
+fieldset {
+    margin-top: 5%;
+    color: black;
+    width: 100%;
+}
+
+.border-lists-skin {
+    border: 0.5px solid rgb(0, 0, 0);
+}
+
+.h2-skin {
```

Comment:
pls do not put tag name in class names. Also check to see if you can use this class on other elements and remove the color: rgb(51,153,255) declaration from them